tensorflow/lite/kernels/atan2_test.cc

```diff
@@ -62,13 +69,77 @@ class Atan2Model : public tflite::SingleOpModel {
   int output_;
 };
 
+template <>
+std::vector<Eigen::half> Atan2Model::GetOutput(
+    const std::vector<Eigen::half>& y, const std::vector<Eigen::half>& x) {
+  PopulateTensor<Eigen::half>(y_, y);
+  PopulateTensor<Eigen::half>(x_, x);
+  Invoke();
+  return ExtractVector<Eigen::half>(output_);
+}
+
+template <>
+std::vector<Eigen::bfloat16> Atan2Model::GetOutput(
+    const std::vector<Eigen::bfloat16>& y,
+    const std::vector<Eigen::bfloat16>& x) {
+  PopulateTensor<Eigen::bfloat16>(y_, y);
+  PopulateTensor<Eigen::bfloat16>(x_, x);
+  Invoke();
+  return ExtractVector<Eigen::bfloat16>(output_);
+}
+
 template <typename Float>
 class Atan2Test : public ::testing::Test {
  public:
   using FloatType = Float;
 };
 
-using TestTypes = ::testing::Types<float, double>;
+using TestTypes = ::testing::Types<float, double, Eigen::half, Eigen::bfloat16>;
+
+TEST(Atan2_test, Atan2_testWorks) {
```

Comment:
What are these tests testing that is not covered under the typed tests? If just ensuring the type coverage works, lets rely on the typed tests for that. If something else, lets use better names for these test cases so they convey the behavior.

Reply:
I've changed it to typed test